### PR TITLE
Adding Cross Tenant DS move

### PR DESCRIPTION
**Pull request by hiaga**
updated help text

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
Please fix the signature issue that defines a default paramter set for Copy-AzRecoveryServicesVault
```
  "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
  "Az.RecoveryServices","Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.CopyAzureRmRecoveryServicesVault","Copy-AzRecoveryServicesVault","1","8510","Cmdlet 'Copy-AzRecoveryServicesVault' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```

File: src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Vault/CopyAzureRmRecoveryServicesVault.cs [hunk 1]
```diff
@@ -31,34 +31,48 @@ public class CopyAzureRmRecoveryServicesVault : RecoveryServicesBackupCmdletBase
     {
         #region Parameters
 
+        internal const string AzureRSVaultDataMoveParameterSet = "AzureRSVaultDataMoveParameterSet";
```
Comment (msJinLei) [inline src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Vault/CopyAzureRmRecoveryServicesVault.cs:34]: Please sepecify a default parameter set for Copy-AzRecoveryServicesVault
```
  "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
  "Az.RecoveryServices","Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.CopyAzureRmRecoveryServicesVault","Copy-AzRecoveryServicesVault","1","8510","Cmdlet 'Copy-AzRecoveryServicesVault' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Vault/CopyAzureRmRecoveryServicesVault.cs (hunk 1 above)
Comment (hiaga) [reply src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Vault/CopyAzureRmRecoveryServicesVault.cs:35]: Specified and re-uploaded the PR

**Comment by hiaga** [comment]
@msJinLei pipeline seems to be stuck

**Review by msJinLei — APPROVED** [review]

**Pull request merged**